projects/map-ol/src/lib/map-ol.service.ts

```diff
@@ -1280,8 +1544,22 @@ export class MapOlService {
       // delete l.custom_layer;
       return layer;
 
+    } else if (l?.mergedLayers?.length) {
+      const layers = l.mergedLayers.map(ml => {
+        /** Set all to visible because the visibility of merge layers cannot be controlled later */
+        ml.visible = true;
+        /** popups are get from the olLayer later so add them */
+        ml.popup = l.popup;
+        /** events are get from the olLayer later so add them */
+        ml.events = l.events;
+        return this.create_layers(ml);
+      });
+      const groupOptions = this.adjustLayerOptions(l, 'custom');
+      (groupOptions as any).layers = layers;
+      const layerGroup = new olLayerGroup(groupOptions);
+      return layerGroup;
     } else {
-      console.log('attribute custom_layer not set on layer type custom!');
+      console.log('attribute custom_layer not set on layer type custom! And it is not a Layer with mergedLayers!');
```

Comment:
as I said [here](https://github.com/dlr-eoc/ukis-frontend-libraries/pull/102/files#r761829361) this attributes (custom_layer and mergedLayers) are a bit confusing to me

Reply:
1caffcf4897a97f83775d80560c41ad42bc68f9c